docs/v0.5/getting-started/docker-for-windows.md

```diff
@@ -135,13 +135,13 @@ Install riff with both the Core and Knative runtimes. To omit or include other r
 
 ```powershell
 helm install projectriff/riff --name riff `
-  --set riff.runtimes.core.enabled=true `
-  --set riff.runtimes.knative.enabled=true `
-  --set riff.runtimes.streaming.enabled=false `
+  --set tags.core-runtime=true \
```

Comment:
line continuation for windows should be ` and not \

Reply:
copy-paste

Reply:
added a commit